lib/models/User.js

```diff
@@ -1,27 +1,31 @@
 const pool = require('../utils/pool');
 
 class User {
-    id;
-    email;
-    firstName;
-    lastName;
-    #passwordHash;
+  id;
+  email;
+  first_name;
+  last_name;
+  #password_hash;
 
-    constructor(row) {
-        this.id = row.id;
-        this.email = row.email;
-        this.firstName = row.firstName;
-        this.lastName = row.lastName;
-        this.#passwordHash = row.password_hash;
-    }
+  constructor(row) {
+    this.id = row.id;
+    this.email = row.email;
+    this.first_name = row.first_name;
+    this.last_name = row.last_name;
```

Comment:
```this``` refers to the object being created by the class and we want the object to have ```camelCase``` keys, ```row``` refers to the JSON object that has been passed to us from postgres and we should expect the keys to be in ```snake_case```